src/components/CommentsList.vue

```
@@ -0,0 +1,66 @@
+<template>
+  <div>
+    <Loader v-if="loading" />
+    <Notification v-if="error" :message="error" type="is-danger" />
+
+    <p v-if="!comments.length && !loading">No comments yet</p>
+
+    <div v-for="comment in comments" :key="comment.id" class="box">
+      <p><strong>{{ comment.name }}</strong> ({{ comment.email }})</p>
+      <p>{{ comment.body }}</p>
+      <button class="button is-small is-danger" @click="deleteComment(comment)">Delete</button>
+    </div>
+
+    <CommentForm :postId="postId" @added="addComment" />
+  </div>
+</template>
+
+<script>
+import Loader from './Loader.vue';
+import Notification from './Notification.vue';
+import CommentForm from './CommentForm.vue';
+
+export default {
+  components: { Loader, Notification, CommentForm },
+  props: { postId: Number },
```

Small prop suggestion: `props: { postId: Number }` is permissive but not declarative about requirement. Consider declaring `postId: { type: Number, required: true }` so the component fails loudly when used without a post id. This helps satisfy the expectation that comments are always associated with a post.